### PR TITLE
Feature/add new records

### PR DESCRIPTION
**Pull request by kuyavinny**

### CODE REVIEW

File: src/controllers/questions.js [hunk 1]
```diff
@@ -4,20 +4,24 @@ import Question from '../models/Question';
 exports.getQuestions = async (req, res) => {
   // eslint-disable-next-line camelcase
   const { product_id, limit, page } = req.query;
-  const results = await Question.queryGetQuestions(product_id, limit, page, true);
-  if (results.status) {
+  const { status, data } = await Question.queryGetQuestions(product_id, limit, page);
```
Comment (Scott-Guinn) [inline src/controllers/questions.js:7]: this destructuring is sleek, I like it

File: src/models/Answer.js [hunk 2]
```diff
@@ -2,67 +2,76 @@ import Photos from './Photos';
...
+  } catch (error) {
+    return { status: false, data: error };
+  }
 };
 
-exports.queryUpdateAnswer = ({ body, helpfulness }, answerId) => {
+exports.queryReportAnswer = (answerId) => {
   const { rows } = query(
     `UPDATE answers
-     SET body = ${body}, a_helpfulness = ${helpfulness}
```
Comment (Scott-Guinn) [inline src/models/Answer.js:64]: be sure to escape these variables

**Review by Scott-Guinn — APPROVED** [review]

File: src/controllers/questions.js (hunk 1 above)
Comment (kuyavinny) [reply src/controllers/questions.js:7]: Thanks!

File: src/models/Answer.js (hunk 2 above)
Comment (kuyavinny) [reply src/models/Answer.js:64]: This is modified in the new PR.

**Pull request merged**